locksmith/src/controllers/v2/og.tsx

```diff
@@ -206,10 +210,15 @@ export const eventOGHandler: RequestHandler = async (request, response) => {
     }
   )
 
+  // OG cannot be SVG
+  const svg = new Resvg(eventBannerSVG)
+  const pngData = svg.render()
```

Comment:
Maybe there is a need to handle a potential error here is the markup is wrong somehow ?

Reply:
That's a good question. The markup is generated by `satori`, so I am not sure it could be invalid, but there could always be issue. I'll add a try/catch